Kickstarter-iOS/Views/Cells/SearchLoadingStateCell.swift

```diff
@@ -0,0 +1,30 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
+import UIKit
+
+internal final class SearchLoadingStateCell: UITableViewCell, ValueCell {
```

Comment:
is this used anymore? it seems the loading indicators are done via the background view and header view?